### PR TITLE
External technologies

### PR DESCRIPTION
**Pull request by hibnico**

### CODE REVIEW

**Review by fabienmifsud — APPROVED** [review]

File: src/main/kotlin/com/saagie/technologies/SaagieTechnologiesUtils.kt [hunk 1]
```diff
@@ -49,9 +50,8 @@ fun modifiedProjects(type: TYPE, subProjects: MutableSet<Project>): Set<Project>
             !pathFile.contains("context.yaml", true) &&
                 !pathFile.contains("technology.yaml", true) &&
                 !pathFile.contains("innerContext.yaml", true) &&
-                !pathFile.contains("metadata.yaml", true) &&
-                !pathFile.contains("build.gradle.kts", true)
```
Comment (skydjol) [inline src/main/kotlin/com/saagie/technologies/SaagieTechnologiesUtils.kt:53]: Why did you delete ?

Comment (hibnico) [reply src/main/kotlin/com/saagie/technologies/SaagieTechnologiesUtils.kt:53]: Because the `build.gradle.kts` is part of what defines how the technology should be built. So it is is modified, probably the technology should be rebuilt.
See for instance: https://github.com/saagie/technologies/blob/master/technologies/job/python/python-2.7/build.gradle.kts#L29

Comment (skydjol) [reply src/main/kotlin/com/saagie/technologies/SaagieTechnologiesUtils.kt:53]: It was an error following the previous modification ?

Comment (hibnico) [reply src/main/kotlin/com/saagie/technologies/SaagieTechnologiesUtils.kt:53]: Yep, I think the test is to broad.
Even if the only modification I do on a technology is modifying the `build.gradle.kts`, to customize the BuildDockerImage task, I would like the technology to be rebuilt.
But indeed, if the yaml are modified, no need to rebuild.
